.github/workflows/packaging.yml

```diff
@@ -56,6 +56,9 @@ jobs:
       && contains(needs.*.result, 'success')
       && !contains(needs.*.result, 'failure')
     steps:
+      - uses: actions/checkout@v2
```

Comment:
```suggestion
      - uses: actions/checkout@v4
```

Reply:
Should we update the other checkout steps as well?